src/ui/components/Account/AccountTeaser.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {SubstrateChainAccount} from 'src/generated/litentryGraphQLTypes';
+import {fireEvent, render} from 'src/testUtils';
+import {AccountTeaser} from './AccountTeaser';
+
+const account = {
+  address: 'FcxNWVy5RESDsErjwyZmPCW6Z8Y3fbfLzmou34YZTrbcraL',
+  balance: '230 DOT',
+  display: 'PolkadotIndia',
+  hasIdentity: false,
+  registration: {
+    judgements: [
+      {
+        registrarIndex: 0,
+        judgement: {
+          isUnknown: false,
+          isFeePaid: true,
+          isReasonable: false,
+          isKnownGood: false,
+          isOutOfDate: false,
+          isLowQuality: false,
+          isErroneous: false,
+        },
+      },
+    ],
+  },
+} as unknown as SubstrateChainAccount;
+
+const accountNavigation = {
+  navigate: jest.fn(),
+};
+
+describe('AddressInfoPreview', () => {
+  it('should render the AccountTeaser component', async () => {
+    const {findByText} = render(<AccountTeaser account={account} onPress={accountNavigation.navigate} />);
```

Comment:
This test is exactly the same as the one in `src/ui/components/Account/Account.test.tsx` Are there any differences in the components for the users?

Reply:
Nope, the same test case gets repeated several times but on different screens. We are providing other same test cases on various screens.